Endless jupnp warnings

My logs are full of:

```
12:23:31 WARN  Could not hydrate device or its services from descriptor: (RemoteDevice) Identity: (RemoteDeviceIdentity) UDN: uuid:bc51db50-7984-44af-947b-a90f538ed0dd, Descriptor: http://192.168.1.2:1649/, Root: true
org.jupnp.binding.xml.DescriptorBindingException: Could not parse device descriptor
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:115)
	at org.jupnp.binding.xml.RecoveringUDA10DeviceDescriptorBinderImpl.describe(RecoveringUDA10DeviceDescriptorBinderImpl.java:54)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:187)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:175)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.run(RetrieveRemoteDescriptors.java:106)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.jupnp.binding.xml.DescriptorBindingException: Could not parse device DOM
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:136)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:110)
	... 7 common frames omitted
Caused by: org.jupnp.model.types.InvalidValueException: Can't parse device type string (namespace/type/version): urn:dial-multiscreen-org:service:dial:1
	at org.jupnp.model.types.DeviceType.valueOf(DeviceType.java:128)
	at org.jupnp.binding.staging.MutableDevice.createDeviceType(MutableDevice.java:82)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:73)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:64)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.buildInstance(UDA10DeviceDescriptorBinderImpl.java:142)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:131)
	... 8 common frames omitted
```

all day every day. I wonder if we can filter these by default

--- a/src/main/java/net/pms/network/mediaserver/jupnp/UmsUpnpServiceConfiguration.java
+++ b/src/main/java/net/pms/network/mediaserver/jupnp/UmsUpnpServiceConfiguration.java
@@ -478,7 +478,7 @@ public class UmsUpnpServiceConfiguration implements UpnpServiceConfiguration {
 
 	private static void resetLoggingMode() {
 		Level mediaServerLevel = CONFIGURATION.isUpnpDebugMediaServer() ? Level.TRACE : Level.INFO;
-		Level basicLevel = CONFIGURATION.isUpnpDebugBasic() ? Level.TRACE : Level.INFO;
+		Level basicLevel = CONFIGURATION.isUpnpDebugBasic() ? Level.TRACE : Level.ERROR;
 		Level fullLevel = CONFIGURATION.isUpnpDebugFull() ? Level.TRACE : Level.ERROR;
 		if (fullLevel == Level.TRACE) {
 			LOGGER.debug("Upnp log mode: full");

